README.md

```diff
@@ -148,5 +148,15 @@ JFrog CLI can be used for a variety of functions with Artifactory, Bintray, Xray
 and has a dedicated set of commands for each product.
 To learn how to use JFrog CLI, please visit the [JFrog CLI User Guide](https://www.jfrog.com/confluence/display/CLI/Welcome+to+JFrog+CLI).
 
+## Using JFrog CLI Docker Image
+The JFrog CLI Docker image may be pulled from Bintray by executing the corresponding Docker command:
```

Comment:
I suggest rephrasing to the following:
*The docker image of JFrog CLI can be pulled from Bintray by running the following command:*